core/testdata/domain.json

```diff
@@ -1,10 +1,10 @@
 {
 	"domain_id": "domain_5577006791947779410",
 	"log": {
-		"tree_id": 4013249521620300357,
+		"tree_id": 2029501983648168255,
 		"tree_state": 1,
 		"tree_type": 1,
-		"hash_strategy": 3,
+		"hash_strategy": 1,
```

Comment:
Are these JSON files marshaled protobufs? If so, using the enum names instead of their values would be nicer.

Reply:
Good idea. Done.

Note that this requires using `jsonpb` to do the encoding rather than `encoding/json`.